tests/acceptance/test_organization_plugin_detail_view.py

```diff
@@ -37,9 +37,8 @@ def test_opsgenie_add_to_project(self):
 
         self.browser.click('[id="react-select-2-option-0"]')
         # check if we got to the configuration page with the form
-        self.browser.wait_until_not(".loading-indicator")
```

Comment:
We should keep this as the condition did succeed (otherwise test would fail here)